### DISCUSSION TITLE
Does setting `allowMethodSpoofing` to true lead to some security or performance issues?

### DISCUSSION THREAD

**Discussion by jitterbux** [adonisjs/core#4102] [General]
In the docs I read that you can set up allowMethodSpoofing to true in config/app.ts.   

Is this change from false to true connected with some security issue or performance issue.

In other words, why isn't is turned on by default ;) ?

**Comment by thetutlage** [comment]
Because not every app might need it (especially pure JSON API servers). So, we expect you to explicitly opt-in.

But there are no security concerns or performance issues with `allowMethodSpoofing`

**Comment by jitterbux** [comment]
OK, great.
